README.md

```diff
@@ -29,10 +25,6 @@ _Optional_ The merge method you would like to use (squash, merge, rebase). Defau
 
 _Optional_ An arbitrary message that you'd like to comment on the PR after it gets auto-merged. This is only useful when you're recieving too much of noise in email and would like to filter mails for PRs that got automatically merged.
 
-### `api-url`
-
-_Optional_ A custom url where the external API which is delegated the task of approving and merging responds.
-
 ### `target`
 
 _Optional_ A flag to only auto-merge updates based on Semantic Versioning. Default to `major` merge. Possible options are:
```

Comment:
there is an inconsistency here and in the next line. default is now any, not major